pubsublite/pscompat/example_test.go

```diff
@@ -97,11 +101,12 @@ func ExamplePublisherClient_Error() {
 		id, err := r.Get(ctx)
 		if err != nil {
 			// TODO: Handle error.
-			if err == pscompat.ErrPublisherStopped {
-				// Prints the fatal error that caused the publisher to terminate.
-				fmt.Printf("Publisher client stopped due to error: %v\n", publisher.Error())
-				break
-			}
+			// NOTE: Once the first publish error occurs, the publisher client will
+			// terminate and subsequent publish results will also have an error.
+
+			// Prints the fatal error that caused the publisher to terminate.
+			fmt.Printf("Publisher client stopped due to error: %v\n", publisher.Error())
```

Comment:
Should users do anything with `err`?

Reply:
They can't do much, besides print/log it to determine the cause of error. In almost all cases (except ErrOversizedMessage), they can just create a new publisher to republish the failed messages. Amended the docs to make this clearer.